jobs/async-upload/tests/test_models.py

```diff
@@ -0,0 +1,273 @@
+"""
+Unit tests for models.py - specifically testing model validation logic.
+"""
+import pytest
+from pydantic import ValidationError
+
+from job.models import ModelConfig, UploadIntent
+
+
+class TestModelConfigValidateModelIds:
+    """Test cases for ModelConfig.validate_model_ids method"""
+
+    def test_create_model_intent_with_all_ids_succeeds(self):
+        """Test that create_model intent succeeds when all required IDs are provided"""
+        config = ModelConfig(
+            upload_intent=UploadIntent.create_model,
+            id="test-model-id",
+            version_id="test-version-id",
+            artifact_id="test-artifact-id"
+        )
+        
+        # Should not raise any validation errors
+        assert config.upload_intent == UploadIntent.create_model
+        assert config.id == "test-model-id"
+        assert config.version_id == "test-version-id"
+        assert config.artifact_id == "test-artifact-id"
+
+    def test_create_model_intent_no_validation(self):
+        """Test that create_model intent has no validation requirements"""
+        # Should succeed even with empty strings since create_model has no validation
+        config = ModelConfig(
+            upload_intent=UploadIntent.create_model,
+            id="",  # Empty string - should be fine
+            version_id="",  # Empty string - should be fine
+            artifact_id=""  # Empty string - should be fine
+        )
+        
+        # Should not raise any validation errors
+        assert config.upload_intent == UploadIntent.create_model
+        assert config.id == ""
+        assert config.version_id == ""
+        assert config.artifact_id == ""
+
+    def test_create_model_intent_with_partial_data_succeeds(self):
+        """Test that create_model intent succeeds with partial data"""
+        config = ModelConfig(
+            upload_intent=UploadIntent.create_model,
+            id="test-model-id",
+            version_id="",  # Empty string - should be fine
+            artifact_id="test-artifact-id"
```

Comment:
analogous question to above, not sure I got the intent of this test? if this is meant to create the triplet, why some IDs for registered_model and model_artifact are supplied?